data/json/mapgen/garage.json

```diff
@@ -0,0 +1,64 @@
+[
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [ "s_garage" ],
+    "weight": 1000,
+    "object": {
+	  "fill_ter": "t_floor",
+      "rows": [
+	    "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "    e         e         ",
+        " ----========--======-- ",
+        " |c.e.........e.......| ",
+        " |c...................| ",
+        " |c...................| ",
+        " |c...................| ",
+        " |c...................| ",
+        " |c...................| ",
+        " |c...................| ",
+        " |....................| ",
+        " |---.................| ",
+        " |..|............HHHHH| ",
+        " |..|............H.bbb| ",
+        " |..*............H....| ",
+        " |..|............$.6b.+ ",
+        " |..|ccccccccccccH.LLL| ",
+        " |----::::::----------| ",
+        "                        ",
+        "                        ",
+        "                        "
+      ],
+      "terrain": {
+        " ": [ "t_grass", "t_grass", "t_grass", "t_dirt" ],
+        "e": "t_gates_mech_control",
+        "=": "t_door_metal_locked",
+        "+": "t_door_c",
+		"*": "t_door_frame",
+		"H": "t_wall_glass",
+		"$": "t_door_glass_c",
+        "6": "t_console_broken",
+        "-": "t_wall",
+        ".": "t_floor",
+        ":": "t_window",
+        "|": "t_wall"
+      },
+      "furniture": {
+        "c": "f_counter",
+        "b": "f_bench",
+        "L": "f_locker"
+      },
+	  "items": {
+        "c": { "item": "mechanics", "chance": 20 },
+        "L": { "item": "clothing_work_set", "chance": 40 }
+	  },	  
```

Comment:
This closing bracket and the opening `"items"` line are both using incorrect indentation